CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * [CHANGE] Ingester: don't update internal "last updated" timestamp of TSDB if tenant only sends invalid samples. This affects how "idle" time is computed. #3727
 * [CHANGE] Require explicit flag `-<prefix>.tls-enabled` to enable TLS in GRPC clients. Previously it was enough to specify a TLS flag to enable TLS validation. #3156
 * [CHANGE] Query-frontend: removed `-querier.split-queries-by-day` (deprecated in Cortex 0.4.0). You should use `-querier.split-queries-by-interval` instead. #3813
+* [CHANGE] Ingester: at startup, if the ingester is already registered in the ring with the `LEAVING` state, it will be switched back to `PENDING` keeping the previous tokens and then follow the normal lifecycler workflow. #3774
```

Comment:
Cortex release 1.8.0 is now in progress. Could you please rebase master and move the CHANGELOG entry under the master / unreleased section?